src/__tests__/components/FileUpload.itest.js

```diff
@@ -5,17 +5,22 @@ import FileUpload from '../../components/FileUpload/FileUpload';
 describe('FileUpload Component', () => {
   const onFileAccepted = jest.fn();
   const files = [{ name: 'file1', size: 100 }];
-  const files2 = [{ name: 'file1', size: 100 }, { name: 'file2', size: 200 }];
+  // const files2 = [{ name: 'file1', size: 100 }, { name: 'file2', size: 200 }];
 
   it('matches snapshot', () => {
     const wrapper = shallow(<FileUpload onFileAccepted={onFileAccepted} files={files} />);
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('handles more than one files', () => {
-    const wrapper = shallow(<FileUpload onFileAccepted={onFileAccepted} files={files2} />);
+  it('enabled by default', () => {
+    const wrapper = shallow(<FileUpload onFileAccepted={onFileAccepted} files={[]} />);
+    expect(wrapper.find('.drop-zone').prop('disabled')).toBe(false);
+  });
+
+  it('disables after a file is added', () => {
+    const wrapper = shallow(<FileUpload onFileAccepted={onFileAccepted} files={files} />);
     expect(wrapper.find('.drop-zone').prop('disabled')).toBe(true);
-    expect(wrapper.find('.title').text()).toBe('Only one file can be selected.');
-    expect(wrapper.find('.sub-title').text()).toBe('');
   });
+
+  it.skip('how do we test the contest of the <p> tag?', () => {});
```

Comment:
might be related to the shallow rendering issue

Reply:
Fixed.